src/popup/router/components/AccountSwitcher.vue

```diff
@@ -68,6 +66,7 @@ export default {
   flex-direction: column;
   border-radius: 0 0 10px 10px;
   padding-top: 32px;
+  margin-bottom: 24px;
```

Comment:
margin with `.buttons` should be 12px (clarify in designs),
without should be 16px